dapps/assets/src/components/digitaltwins/templates/car.json

```diff
@@ -205,7 +205,7 @@
             "name": "Specifications",
             "owner": {
               "description": "Details of the owner",
-              "name": "Name",
+              "name": "Owner",
```

Comment:
I think this part was ok with "Name" ... should check where the explicit change was needed.

Reply:
a this "Name" is the name of the "owner" property ... really confusing :D than it's cool. Just update also here, to keep in sync, till we pull it soon™️ directly from this repo: https://github.com/evannetwork/twin-templates